include/bluetooth/conn.h

```diff
@@ -162,6 +171,8 @@ struct bt_conn_info {
 		struct bt_conn_le_info le;
 
 		struct bt_conn_br_info br;
+
+		struct bt_conn_rv_info rv;
```

Comment:
This is overlapping with le_info since it is a union, we need to store this info elsewhere, perhaps a dedicated API that does take a pointer to store this info and block while doing the HCI command, though in case of multiple thread trying to access it may have to be a lock/flag.

Reply:
@Vudentz Could I just move the remote version struct outside of the union and wrap it in a "#if defined"?